docker/api.dockerfile

```
@@ -19,11 +19,14 @@ RUN apk add --no-cache libc6-compat
 
 WORKDIR /app
 
+RUN npm install -g corepack
+RUN corepack enable
+RUN corepack prepare pnpm@latest --activate
```

```suggestion
RUN npm install -g corepack@latest
RUN corepack enable
```
I think `corepack prepare` is not appropriate.